Podcast/SearchTableViewController.swift

```diff
@@ -43,38 +40,43 @@ class SearchTableViewController: UITableViewController, SearchEpisodeTableViewCe
     let cellIdentifiersClasses: [SearchType: (String, AnyClass)] =
         [.episodes: ("EpisodeCell", SearchEpisodeTableViewCell.self),
          .series: ("SeriesCell", SearchSeriesTableViewCell.self),
-         .people: ("PeopleCell", SearchPeopleTableViewCell.self),
-         .tags: ("TagCell", SearchTagTableViewCell.self)]
+         .people: ("PeopleCell", SearchPeopleTableViewCell.self)]
+    
     let cellHeights: [SearchType: CGFloat] =
         [.episodes: 84,
          .series: 95,
-         .people: 76,
-         .tags: 53]
+         .people: 76]
     
     var searchResults: [SearchType: [Any]] = [
         .episodes: [],
         .series: [],
-        .people: [],
-        .tags: []]
+        .people: []]
     
     var cellDelegate: SearchTableViewControllerDelegate?
     var loadingIndicatorView: NVActivityIndicatorView?
     
+    var continueInfiniteScroll: Bool = true
     var currentlyPlayingIndexPath: IndexPath?
     
     override func viewDidLoad() {
         super.viewDidLoad()
-        guard let appDelegate = UIApplication.shared.delegate as? AppDelegate else { return }
         guard let (cellIdentifier, cellClass) = cellIdentifiersClasses[searchType] else { return }
         tableView.register(cellClass, forCellReuseIdentifier: cellIdentifier)
```

Comment:
Won't this only register one cell type on viewDidLoad? If I switch tabs mid search what happens?

Reply:
Nope! we instantiate a different SearchTableViewController for each search type which is part of our paggedviewcontrollers viewcontrollers

Reply:
Ahh gotcha.